pypeit/core/procimg.py

```diff
@@ -775,7 +777,7 @@ def variance_frame(datasec_img, sciframe, gain, ronoise, numamplifiers=1, darkcu
     # ToDO JFH: I would just add the darkcurrent here into the effective read noise image
     # The effective read noise (variance image)
     if rnoise is None:
-        rnoise = rn_frame(datasec_img, gain, ronoise, numamplifiers=numamplifiers)
+        rnoise = rn_frame(datasec_img, gain, ronoise)
```

Comment:
I'm a bit torn on this.  I'm generally of the mind that explicit is better than implicit, but the definition of `datasec_img` is such that it must contain `numamplifiers`. So I think I'd be okay, if we remove `numamplifiers` as an argument to both `rn_frame` and `variance_frame`.  Having said that, if `numamplifiers` is passed to `variance_frame`, it should also be passed on to `rn_frame`; and we should set the default of `numamplifiers` to be the same for both `rn_frame` and `variance_frame`.

Reply:
Sounds good to me.